### DISCUSSION TITLE
Snoverchill

### DISCUSSION THREAD

**Discussion by Tskk-dev** [skyline-emu/skyline#921] [Game Requests]
This game is the magnum opus of eSports so I was wondering if it can be added 🙏
![liptrick](https://user-images.githubusercontent.com/96981837/162007891-8cea6197-5916-43e1-a96b-f5c628ec0ecc.gif)

**Comment by PixelyIon** [comment]
It cannot, the graphics are too complex for the measly Switch. It may however be possible for the esteemed Snoverchill developer @riperiperi to add support for the Switch, we can strive for support of this title in accordance to development of such a port.

**Comment by riperiperi** [comment]
sorry, the nintendo switch browser applet is not powerful enough to support snoverchill. it must be ported to switch directly first.

**Reply by Tskk-dev** [reply]
Skill issue

**Comment by Swizzlefowl** [comment]
Truly
